### DISCUSSION TITLE
Variables / input fields for shop packs

### DISCUSSION THREAD

**Discussion by DieselEffect** [gameserverapp/Platform#402] [Suggestions]
A fillable form where tasks can be completed upon it's submission with or without the contents of the form as usable parameters.

Examples may be:

1) A community wants to conduct a survey to ask players where they found the community, whether via friend invite or google search or what, and wants to reward the player for with tokens taking the survey.

2) A community wants to give out dinos as an event but wants the player to be able to choose the color regions of the dino, so the form asks what color ID for each region and factors that into the spawn code once the form is submitted.

**Comment by mvaessen** [comment]
Changing the title and description a little bit: add input field to shop packs so people can order something from the shop and enter custom stuff via input fields

**Comment by GeckoGamer** [comment]
would be nice if we could add custom variables, for example. {gender}, or {level} to allow players to select a variable when ordering packs.

scriptcommand SpawnDinoInBall -p={characterid} -t=/Game/PrimalEarth/Dinos/Rex/Rex_Character_BP.Rex_Character_BP -l=315 -f={gender:0:1} -i=1 -a=1 -c=1

maybe with the ability to make our own variables to make it easier.
